cyclops-ui/src/components/k8s-resources/DefaultResource.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useCallback, useEffect, useState } from "react";
+import { Alert } from "antd";
+import axios from "axios";
+import { mapResponseError } from "../../utils/api/errors";
+import ResourceList from "./ResourceList/ResourceList";
+
+interface Props {
+  group: string;
+  version: string;
+  kind: string;
+  name: string;
+  namespace: string;
+  onResourceDelete: () => void;
+}
+
+const DefaultResource = ({
+  group,
+  version,
+  kind,
+  name,
+  namespace,
+  onResourceDelete,
+}: Props) => {
+  const [resource, setResource] = useState({
+    children: [],
+  });
+  const [error, setError] = useState({
+    message: "",
+    description: "",
+  });
+  const [loadResources, setLoadResources] = useState(false);
+
+  const fetchResource = useCallback(() => {
+    axios
+      .get(`/api/resources`, {
+        params: {
+          group: group,
+          version: version,
+          kind: kind,
+          name: name,
+          namespace: namespace,
+        },
+      })
+      .then((res) => {
+        setResource(res.data);
+      })
+      .catch((error) => {
+        setError(mapResponseError(error));
+      })
+      .finally(() => {
+        setLoadResources(true);
+      });
+  }, [name, namespace, group, version, kind]);
+
+  useEffect(() => {
+    fetchResource();
+
+    const interval = setInterval(() => fetchResource(), 15000);
```

Comment:
The interval for the CRD to fetch resources is set to 15s, but for the Module to fetch its resources is set to 10s